### ISSUE TITLE
Update docs regarding OpenMPI requirements of FDS 6.0.1

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2110]
```
Please complete the following:

Version: FDS 6.0.1
SVN Revision Number: 17534
Compile Date: Tue, 26 Nov 2013


Running Ubuntu 13.10 I followed those docs and decided to take the package provided
by Ubuntu
* https://code.google.com/p/fds-smv/wiki/Running_FDS_MPI_on_Linux

Since I wanted to use a packaged version of OpenMPI I did these two steps 
* sudo apt-get install openmpi-bin
* install FDS 6.0.1


mpirun is available, but running fds_mpi resulted in error:

/opt/FDS/FDS6/bin/fds_mpi: error while loading shared libraries: libmpi_f77.so.1: cannot
open shared object file: No such file or directory


Googling after that, almost all results talk about LD_LIBRARY_PATH. This is kind of
misleading to potential FDS user like me.

Long story short. Ubuntu 13.10 comes with openmpi-1.4.5, Ubuntu 14.04 comes with openmpi-1.6.5

Looking at these pages, made me connect the 6.0.1 changelog and my problem
* http://packages.ubunut.com/de/saucy/openmpi-bin

I downloaded FDS 6.0.0 and it worked out-of-the-box. Please make this information more
visible.

Now I know, that some trying to run FDS in parallel just uses Ubuntu 14.04, installs
openmpi-bin using package manager and FDS >= 6.0.1 and is ready to go after a couple
of minutes. Sounds like a success story to me.
```

Original issue reported on code.google.com by `sascha.gottfried` on 2014-05-08 14:04:12

**Comment by gforney** [comment]
```
Thanks for the information. We'll make sure the next release incorporates this info.
```

Original issue reported on code.google.com by `mcgratta` on 2014-05-08 14:06:33

**Comment by gforney** [comment]
```
Our wiki you pointed to \
https://code.google.com/p/fds-smv/wiki/Running_FDS_MPI_on_Linux
also lists sudo apt-get install openmpi-bin as an option. I assume that you had not
defined your LD_LIBRARY_PATH correctly.

The fds installer assumes that the mpi packages are loaded at /shared/openmpi_64 and
defines the LD_LIBRARY_PATH variable for this.  I'll update our wiki to be more explicit
about defining the LD_LIBRARY_PATH variable if the libraries are put somewyhere else
and add a link to the web page you indicated for those that use Ubuntu.  

Things would be a lot easier if we only supported one Linux distribution but since
we have to support others our instructions need to be more general.

Thanks for the information you provided
```

Original issue reported on code.google.com by `gforney` on 2014-05-08 16:40:05

**Comment by gforney** [comment]
```
Glenn,
this issue is not about setting LD_LIBRARY_PATH, it is about that FDS 6.0.1 needs OpenMPI
1.6.5 and I did not realized that Ubuntu 13.10 is providing OpenMPI 1.4.5. I just wanted
to give feedback of my expericence from today. For now I stick with FDS 6.0.0 until
I upgrade to Ubuntu 14.04 or choose one from the two alternatives to install OpenMPI
binaries or compile from source.

Shared library dependencies of fds_mpi(6.0.1). These two missing dependencies made
me google for term libmpi_f77.so.1

gottfried@sascha-Latitude-XT2:~$ ldd /opt/FDS/FDS6/bin/fds_mpi 
    linux-vdso.so.1 =>  (0x00007f00da0fe000)
    libmpi_f77.so.1 => not found
    libmpi.so.1 => not found
    libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f00d9ee2000)
    libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f00d9bde000)
    libnuma.so.1 => /usr/lib/libnuma.so.1 (0x00007f00d99d3000)
    librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f00d97ca000)
    libnsl.so.1 => /lib/x86_64-linux-gnu/libnsl.so.1 (0x00007f00d95b0000)
    libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f00d93ad000)
    libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f00d918f000)
    libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f00d8dc7000)
    libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f00d8bb1000)
    /lib64/ld-linux-x86-64.so.2 (0x00007f00da100000)


----------------------------------------------------------------

Shared library dependencies of fds_mpi(6.0.0):

gottfried@sascha-Latitude-XT2:~$ ldd /home/gottfried/FDS/FDS6/bin/fds_mpi 
    linux-vdso.so.1 =>  (0x00007fff61bfe000)
    libmpi_f77.so.0 => /usr/lib/libmpi_f77.so.0 (0x00007f95682e4000)
    libmpi.so.0 => /usr/lib/libmpi.so.0 (0x00007f9568032000)
    libopen-rte.so.0 => /usr/lib/libopen-rte.so.0 (0x00007f9567de3000)
    libopen-pal.so.0 => /usr/lib/libopen-pal.so.0 (0x00007f9567b8a000)
    libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f9567986000)
    libnsl.so.1 => /lib/x86_64-linux-gnu/libnsl.so.1 (0x00007f956776b000)
    libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f9567568000)
    libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f9567264000)
    libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f9567046000)
    libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f9566c7e000)
    libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f9566a68000)
    /lib64/ld-linux-x86-64.so.2 (0x00007f956852e000)
gottfried@sascha-Latitude-XT2:~$ /home/gottfried/FDS/FDS6/bin/fds_mpi 
Process   0 of   0 is running on sascha-Latitude-XT2

Fire Dynamics Simulator

Version: FDS 6.0.0; MPI Enabled; OpenMP Disabled
SVN Revision Number: 17279
Compile Date: Sun, 03 Nov 2013

Consult FDS Users Guide Chapter, Running FDS, for further instructions.

Hit Enter to Escape...


-----------------------------------------------
Users following https://code.google.com/p/fds-smv/wiki/Running_FDS_MPI_on_Linux#Installing_Open_MPI_using_a_package_manager
and provide OpenMPI using their distro package manager apparently do not need to change
LD_LIBRARY_PATH. I manually removed /shared/openmpi_64 from LD_LIBRARY_PATH to underpin
that. fds_mpi still runs fine.

gottfried@sascha-Latitude-XT2:~/FDS/FDS6/bin/LIB64$ echo $LD_LIBRARY_PATH 
/home/gottfried/FDS/FDS6/bin/LIB64


---------------------------------------------
Just as Kevin said, any of the next releases should incorporates this info.
```

Original issue reported on code.google.com by `sascha.gottfried` on 2014-05-08 20:20:29

**Comment by gforney** [comment]
```
I was able to get fds 6.1 (fds built from latest source) working with the ubuntu 13.10
installer after typing the following two commands.  Apparently ubuntu failed to include
the .1 version of these files.

sudo ln -s /usr/lib/libmpi_f77.so.0 /usr/lib/libmpi_f77.so.1
sudo ln -s /usr/lib/libmpi.so.0 /usr/lib/libmpi.so.1

You also need to add /usr/bin to the PATH (if it is not there already) since that is
where ubuntu put the mpirun command (necessary if you have the Intel compilers installed
since Intel puts an mpirun command in its directory tree which is not compatible)

Also note that fds_mpi should work with 1.4.x and 1.6.x versions of the openmpi libraries.
 The issues you were seeing were because of missing files not library incompatibilities.
 Kevin can correct me if I am mistaken.

For future reference, I also tried installing the pre-built openmpi libraries (Alternative
1 at https://code.google.com/p/fds-smv/wiki/Running_FDS_MPI_on_Linux?ts=1399645185&updated=Running_FDS_MPI_on_Linux
) and this worked also.

I added some notes to the mpi documentation regarding the missing files.  Thanks for
the catch.
```

Original issue reported on code.google.com by `gforney` on 2014-05-09 14:31:37

**Comment by gforney** [comment]
```
Glenn, 
thanks a lot for investigating further. That made it a clear win-win situation. Now
I can work with latest release. I would not have tried to create symbolic links, because
actually I do not understand shared library naming conventions and stuff.

For documentation purposes - Running ldd again after creating these two symlinks:

gottfried@sascha-Latitude-XT2:/opt/FDS/FDS6/lib$ ldd /opt/FDS/FDS6/bin/fds_mpi 
    linux-vdso.so.1 =>  (0x00007fff18dfe000)
    libmpi_f77.so.1 (0x00007f4c965f7000)
    libmpi.so.1 (0x00007f4c96344000)
    libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f4c9612b000)
    libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f4c95e27000)
    libnuma.so.1 => /usr/lib/libnuma.so.1 (0x00007f4c95c1c000)
    librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f4c95a13000)
    libnsl.so.1 => /lib/x86_64-linux-gnu/libnsl.so.1 (0x00007f4c957f9000)
    libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f4c955f6000)
    libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f4c953d8000)
    libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f4c95010000)
    libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f4c94dfa000)
    libopen-rte.so.0 => /usr/lib/libopen-rte.so.0 (0x00007f4c94bab000)
    /lib64/ld-linux-x86-64.so.2 (0x00007f4c9682c000)
    libopen-pal.so.0 => /usr/lib/libopen-pal.so.0 (0x00007f4c94952000)


gottfried@sascha-Latitude-XT2:~$ mpirun -np 2 -wdir /opt/firesim/jobs/whirl2 /opt/FDS/FDS6/bin/fds_mpi
/opt/firesim/jobs/whirl2/fire_whirl_pool.fds

Process   0 of   1 is running on sascha-Latitude-XT2
Process   1 of   1 is running on sascha-Latitude-XT2
Mesh   1 is assigned to Process   0
Mesh   2 is assigned to Process   1

 Fire Dynamics Simulator

 Compilation Date : Tue, 26 Nov 2013
 Current Date     : May 10, 2014  11:56:26

 Version: FDS 6.0.1; MPI Enabled; OpenMP Disabled
 SVN Revision No. : 17534

 Job TITLE        : UT Fire Whirl
 Job ID string    : fire_whirl_pool


STOP: Set-up only
```

Original issue reported on code.google.com by `sascha.gottfried` on 2014-05-10 10:19:44

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `gforney` on 2014-05-15 17:00:19
